tests/test_lookml.py

```diff
@@ -705,6 +706,24 @@ def msg_glean_probes():
     ]
 
 
+@contextlib.contextmanager
+def pool_started():
+    """
+    Needed when mocking objects that are used within a multiprocessing pool.
+
+    The default start method when using multiprocessing is spawn on MacOS and Windows.
+    If using fork, your processes are forked in the current state, including the mocking,
+    while with using spawn, a new Python interpreter is launched, where the mock will not work.
+    The start method needs to be changed to fork for the mocks to work across processes.
+    """
+    start_method = multiprocessing.get_start_method()
+    try:
+        multiprocessing.set_start_method("fork", force=True)
+        yield
+    finally:
+        multiprocessing.set_start_method(start_method, force=True)
```

Comment:
The [Python docs](https://docs.python.org/3.10/library/multiprocessing.html#contexts-and-start-methods) say "`set_start_method()` should not be used more than once in the program", but it doesn't seem like their recommended alternative of using [`get_context()`](https://docs.python.org/3.10/library/multiprocessing.html#multiprocessing.get_context) would be as easy to use because it isn't a context manager that automatically takes precedence, we'd have to change the code to have a configurable `multiprocessing` parameter and use/override that.